pkg/storaged/block/format-dialog.jsx

```diff
@@ -597,7 +633,10 @@ function format_dialog_internal(client, path, start, size, enable_dos_extended,
                         return (client.wait_for(() => block_fsys_for_block(path))
                                 .then(block_fsys => client.mount_at(client.blocks[block_fsys.path],
                                                                     mount_point)));
-                    if (is_encrypted(vals) && is_filesystem(vals) && !mount_now)
+                    if (type == "swap" && mount_now)
+                        return (client.wait_for(() => block_swap_for_block(path))
+                                .then(block_swap => block_swap.Start({})));
+                    if (is_encrypted(vals) && (is_filesystem(vals) || type == "swap") && !mount_now)
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19843-20240118-104422-e6fc1c3f-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)